pandas/core/construction.py

```diff
@@ -533,6 +533,10 @@ def _try_cast(
     if isinstance(dtype, ExtensionDtype) and dtype.kind != "M":
         # create an extension array from its dtype
         # DatetimeTZ case needs to go through maybe_cast_to_datetime
+
+        if lib.is_scalar(arr):
```

Comment:
we could look into how to identify a collection that could be considered a 'scalar' for some EA, eg JSONDtype. although I think out-of-scope for the issue that this PR attempts to fix (i.e. IntegerArray, where the scalars are scalars)

Reply:
rather than this I would call: `construct_1d_arraylike_from_scalar`

but I wouldn't do this right here, rather on L453, e.g. add an elif is_scalar(data)

Reply:
that's option 3 in https://github.com/pandas-dev/pandas/pull/33846#issuecomment-621736781

do this just for EA types and keep the code path the same for non-EA types?

Reply:
no this will work generically

Reply:
kk